test/test811.js

```diff
@@ -102,7 +102,7 @@ describe('Test 811 - String / Number objects', function () {
 	});
 
 	it('5. Where In', function (done) {
-		var t1 = [{ID: new String('s1')}, {ID: new String('s2')}, {ID: new String('s3')}];
+		var t1 = [{ID: "s1"}, {ID: "s2"}, {ID: "s3"}];
```

Comment:
This is a breaking change. We need to seperate that out of this PR

Reply:
I will look into it, maybe the implementation using sets can be adjusted to also allow for using new String.